Dockerfile

```diff
@@ -4,7 +4,8 @@ FROM google/cloud-sdk:295.0.0
 ENV HOME=/
 
 # install python3 to make that also available - modified from https://tecadmin.net/install-python-3-7-on-ubuntu-linuxmint/
-RUN apt-get install -y build-essential checkinstall \
+RUN apt-get update \
```

Comment:
I found that when building a fresh image, the build was broken here, likely because we're using an old `apt-get`.  this probably hasn't been caught because builds of `latest` are using caching so it rarely has to rebuild this layer.  adding `apt-get update` seemed to fix things